### ISSUE TITLE
Idea: support syntax formatting in markdown's "talon code blocks"

### ISSUE THREAD

**Issue by saidelike** [mrob95/vscode-TalonScript#3]
Not sure if that would be easy to do, but assuming we are editing a markdown file, and document some talon code in a code block, it would be useful to have syntax highlighting of the talon code in these blocks. Similarly to any other language blocks like C, Python, etc.

```
This is an example for opening a new window:

~~~talon
new window: key(ctrl-shift-n)
~~~

```

**Comment by mirober** [comment]
Added in https://github.com/mrob95/vscode-TalonScript/commit/8a9591bbaa9e65c2500685fe6780cb6f917c3728

![image](https://github.com/mrob95/vscode-TalonScript/assets/42875462/94a80fad-f9f6-4c74-852c-f902dd440cc5)

released in v3.2.0

thanks for the suggestion!

**Comment by saidelike** [comment]
Nice, confirmed it works!
